libusb/include/usbdriver.h

```diff
@@ -23,6 +23,10 @@
 #include <sys/msg.h>
 #include <posix/idtree.h>
 
+#define USB_DRVNAME_MAX 10
+#define USB_DEVPATH_MAX 32  /* intentionally not using PATH_MAX, as we expect devices from /dev */
```

Comment:
I don't see this being used anywhere.

Reply:
Well, I think the idea was to use this for `devPath` in this repo and `path` in USB device drivers.